packages/drivers/odsp-driver/src/odspDocumentStorageManager.ts

```diff
@@ -221,6 +221,14 @@ export class OdspDocumentStorageService implements IDocumentStorageService {
     }
 
     public async read(blobid: string): Promise<string> {
+        return this.readWithEncodingOutput(blobid, "base64");
+    }
+
+    public async readString(blobid: string): Promise<string> {
+        return this.readWithEncodingOutput(blobid, "utf8");
```

Comment:
if possible, we should not use "utf8", but rather use "string".
The reason being - we actually do not know (for sure) how strings are represented in JS engine memory, as well as how they are encoded on the wire (if we use stirngs) - protocol for json payload auto-detects encoding based on first 4 bytes of input. So it's really a string on both ends. utf8 is misleading across all of our layers for this reason :(